.agents/skills/vue.skill.md

```
@@ -0,0 +1,253 @@
+      }})
+  }
+```
+
+## 模板规范
+
+1. 必须使用 vue2, vue3同时兼容的模板语法。
+2. 必须是单根节点
+3. 不允许 v-if / v-for在同一个节点上， 建议使用 <template> 来使用 v-if / v-for
+4. 不允许使用 `id` 等属性
+5. 允许使用 Teleport 组件， 但必须从 '@opentiny/vue-common' 包中导入。
+6. 模板中可以使用 `a函数`，比如： ` v-bind="a($attrs, ['class', 'style', 'title', 'id'], true)"` ， `a 函数`的意思是从$attrs上过滤出一些属性绑定到元素上。 最后一个参数为true的话，表示这些属性要保留下来，如果为false，则表示这些属性要过滤掉，其它的属性才保留下来。
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)